flow360/__init__.py

```diff
@@ -144,3 +145,130 @@
 from .flags import Flags
 from .user_config import UserConfig
 from .version import __version__
+
```

Comment:
ruff complains import but not used for all these __init__.py files.

Reply:
I think we should have an __all__ variable listing all imported types: https://github.com/astral-sh/ruff/issues/484

It seems like ruff should be able to fix those if ran with the --fix flag.

Reply:
see here: https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/__init__.py

Reply:
I don't know if ruff auto fix them once we put __all__ in __init__.py. I manual fixed these.